### ISSUE TITLE
how do i make it azerty?

### ISSUE THREAD

**Issue by Pluzas** [kregerl/BakkesKBMOverlay#12]
i have azerty keyboard

**Comment by tosu-dev** [comment]
I went to the code and a just change some lines to make it "azerty", it is pretty simple to understand if you read the code. If I still have this version, I can give it to you if you want.

**Comment by Pluzas** [comment]
Yes please, can you link me the original code you went to aswell?

**Comment by Pluzas** [comment]
when bro?

**Comment by Pluzas** [comment]
i found the source code and i changed it to azerty, hopefully, but idk how to make it appear in game, how to save my version of the code as dll plugin in bakkesmod?

**Comment by Pluzas** [comment]
bro died

**Comment by tosu-dev** [comment]
Sorry I literraly died (no notification). 
You can see in this PR the code to make it AZERTY : https://github.com/kregerl/BakkesKBMOverlay/pull/13/files
To save your version, you have to build it, it will create the dll file. This link might help you : https://www.reddit.com/r/bakkesmod/comments/ejya18/looking_for_a_tutorial/
